tests/mq_protocol_tests/main.go

```diff
@@ -113,7 +113,7 @@ func testMySQL() {
 
 func testMySQLWithCheckingOldValue() {
 	env := mysql.NewDockerEnv(*dockerComposeFile)
-	env.DockerComposeOperator.ExecEnv = []string{"GO_FAILPOINTS=github.com/pingcap/tiflow/cdc/sink/SimpleMySQLSinkTester=return(ture)"}
```

Comment:
Find `return(ture)` can also trigger failpoint, interesting.

Reply:
`return` is enough for failpoint which doesn't use the value